__tests__/app.test.js

```diff
@@ -70,3 +69,45 @@ describe("2. GET /api/articles", () => {
       });
   });
 });
+
+describe("3. GET /api/articles/:article_id", () => {
+  it("200: should respond with the article object containing the corresponding article_id", () => {
+    const articleId = 1;
+    return request(app)
+      .get(`/api/articles/${articleId}`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
```

Comment:
We should use objectContaining or toMatchObject as toEqual will prevent us to add more properties in the future 🙂